{"uuid": "53c8b8dd-b117-4df1-89eb-b1a8a9041f90", "children": ["2aa2e4ae-0450-4368-a68e-d21fea4df489", "dee74eab-661d-47ef-97c7-bce09a101d20", "6085858a-0b50-4e14-b342-c467f227d218", "714a9e24-458d-4d14-831b-fe5d33ce50be", "4f3127a4-9179-4469-92a0-e0d0b362e14b", "3e90f54d-0512-4ec6-989a-b240459bb3d5"], "befores": [{"name": "setup_profiles", "status": "failed", "statusDetails": {"message": "Failed: AP latest and active are different\n", "trace": "  File \"/usr/local/lib/python3.8/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/wlan-testing/tests/e2e/interOp/conftest.py\", line 228, in setup_profiles\n    return_var = fixtures_ver.setup_profiles(request, param, setup_controller, testbed, get_equipment_ref,\n  File \"/wlan-testing/tests/fixtures_2x.py\", line 571, in setup_profiles\n    pytest.fail(\"AP latest and active are different\")\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/outcomes.py\", line 153, in fail\n    raise Failed(msg=msg, pytrace=pytrace)\n"}, "attachments": [{"name": "Active Config: ", "source": "b3f92905-5718-4256-bb46-a667eddeb2e6-attachment.attach"}, {"name": "Latest Config: ", "source": "8c4321b5-cb81-4aff-a1a0-24ea9fe8e10f-attachment.attach"}, {"name": "FAILURE: AP Logs: ", "source": "a6644be5-a85f-43ed-a0a4-7143d489c420-attachment.attach"}], "start": 1639084763168, "stop": 1639085085174}], "start": 1639084763168, "stop": 1639085087081}